sky/clouds/service_catalog/data_fetchers/fetch_fluidstack.py

```diff
@@ -149,6 +178,7 @@
 GPU_MAP = {
     'H100_PCIE_80GB': 'H100',
     'H100_NVLINK_80GB': 'H100',
+    'H100_SXM5_80GB': 'H100',
```

Comment:
We do have some clouds mapping to `H100-SXM`. Maybe fine to map it to H100-SXM as well.

Reply:
Fixed.